### PR TITLE
COM-2655: Edit Quote Page API integration part 1

### PR DESCRIPTION
**Pull request by sagarraut0007**
This story have covered api integration for following fields. Only following fields will be updated as part of this story.
1) Quote Details
        Name
        Expiration Date and Submitted By (seller only)
2) Quote Summary
        Add/Remove Items
        Update Quote item Quantity, Price and Fulfillment Info
        Update Quote Adjustments

### CODE REVIEW

File: labels/de-DE.json [hunk 1]
```diff
@@ -297,8 +302,8 @@
   "mobileOrderConfirmationMessage": "{0} Bestellbenachrichtigungen. Wir werden Ihnen eine SMS senden, wenn sich Ihr Bestellstatus ändert. Die Meldungsfrequenz kann variieren. Antworte {1}, um dich abzumelden. Es können Nachrichten- und Datentarife gelten",
   "mobileOrderOptInMessage": "Vielen Dank, dass Sie sich für SMS-Benachrichtigungen für Ihre Curbside-Bestellung angemeldet haben. Wir werden Sie benachrichtigen, wenn Ihre Bestellung zur Abholung bereit ist.",
   "mobilePickLocation": "Ramasser / Expédier de:",
-  "mobilePickupReminder":"Erinnerung - Artikel in Ihrer Bestellung {0} können abgeholt werden. Überprüfen Sie die Bestellung unter {1}",
-  "mobileReadyForPickup":"Artikel in Ihrer Bestellung {0} können um {1} abgeholt werden. Überprüfen Sie die Bestellung unter {2}",
+  "mobilePickupReminder": "Erinnerung - Artikel in Ihrer Bestellung {0} können abgeholt werden. Überprüfen Sie die Bestellung unter {1}",
```
Comment (ghost) [inline labels/de-DE.json]: I think I've reverted this whitespace change around 3 times now in different PRs. We really need to disable format on save and be careful not to commit changes we don't need (even whitespace). In private repos I'd let this slide but this is public so would rather we don't.

Same with `orderComments` and `orderPickupReminderTitle`.

File: labels/en-US.json [hunk 2]
```diff
@@ -369,8 +374,8 @@
   "mobileOrderConfirmationMessage": "{0} Order Alerts. We will text you when your order status changes. Msg freq may vary. Reply {1} to opt out. Msg&Data rates may apply",
   "mobileOrderOptInMessage": "Thank you for opting in to SMS alerts for your Curbside order. We will notify you when your order is ready for pickup.",
   "mobilePickLocation": "Pickup from/Ship from:",
-  "mobilePickupReminder":"Reminder - Items in your order {0} are ready for pickup. Review the order at {1}",
-  "mobileReadyForPickup":"Items in your order {0} are ready for pickup at {1}. Review the order at {2}",
+  "mobilePickupReminder": "Reminder - Items in your order {0} are ready for pickup. Review the order at {1}",
```
Comment (ghost) [inline labels/en-US.json]: Same as above whitespace comment.

File: scripts/modules/b2b-account/quotes.js [hunk 3]
```diff
@@ -315,7 +315,13 @@ define([
         },
         editQuote: function (e, row) {
             var quoteId = row.get('id');
-            window.location = '/myaccount/quote/' + quoteId + '/edit';
+            var isSalesRep = require.mozuData('user').isSalesRep;
+            if (isSalesRep) {
+                window.location = '/selleraccount/quote/' + quoteId + '/edit';
+            }
+            else {
```
Comment (ghost) [inline scripts/modules/b2b-account/quotes.js]: Minor but the `else` should be on the same line as the `}` in javascript code.

**Review by ghost — APPROVED** [review]
Didn't check every line in detail (no time) but overall code looks clean and readable. See my comments for some minor changes.

**Comment by sagarraut0007** [comment]
@michaelsadlonkibo , Fixed code review comments

**Pull request merged**